src/pages/contributors/kevin-martens.adoc

```diff
@@ -0,0 +1,70 @@
+= Kevin Martens
+:page-name: Kevin Martens
+:page-linkedin: kevinmartens27
+:page-twitter: 
+:page-github: kmartens27
+:page-email: 
+:page-image: avatar/kevin-martens.jpeg
+:page-pronouns: He/Him/His
+:page-location: Boston, Massachusetts, USA
+:page-firstcommit: 2022
+:page-datepublished: 2024-05-08
+:page-featured: true
+:page-intro: Kevin Martens is a technical content developer and the Documentation Officer for Jenkins. He is a musician at heart, with a love for education and writing. Despite joining the Jenkins project somewhat recently, Kevin's presence is felt whether leading documentation office hours or providing reviews for contributors.
+
+Outside of documentation, Kevin can usually be found picking out a record to listen to or playing guitar along to his favorite artists.
+He's also a fan of all kinds of games, favoring card and video games the most.
+On top of all that, Kevin loves to explore around New England with his wife and then come home to their four cats.
+
+== What is your background prior to contributing to Jenkins?
+
+My first experiences with open source involved game emulation, but I was strictly a user in these cases.
+Eventually, when I went to college I utilized open-source versions of documentation and presentation tools since there was no cost involved.
+I went to school for communication & education, going on to teach after graduation.
+However, during this time I learned that despite my passion, teaching was not going to be my career.
+
+After realizing this, I spent time in different areas of the tech space, doing support and training on various hardware and software platforms.
+Ultimately, these roles were not as fulfilling as I had hoped.
+Even though I enjoyed helping users figure out their issues, I felt like something was missing.
+Thanks to my wife's encouragement, I decided to break out from what I had been doing and apply for roles that would utilize my writing skills and background.
+
```

Comment:
I would add a line about how you got to CloudBees/Jenkins. Something like:
```suggestion
Eventually, I was hired by CloudBees, and was trilled to be assigned to the Jenkins project!
```
I am only using "Eventually" as a transition word. Please feel free to update with a fitting adverb.